src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java

```diff
@@ -26,7 +26,7 @@
     SLOT_SELECTORS.put(TOP_BOXAD, "div[id$='top_boxad_0__container__']");
     SLOT_SELECTORS.put(INCONTENT_BOXAD, "div[id$='incontent_boxad_0__container__']");
     SLOT_SELECTORS.put(BOTTOM_BOXAD, "div[id$='bottom_boxad_0__container__']");
-    SLOT_SELECTORS.put(FEED_BOXAD, "div[id$='feed_boxad_0__container__']");
+    SLOT_SELECTORS.put(FEED_BOXAD, "div[id$='google_ads_iframe_/5441/wka1a.PF/feed_boxad/desktop/ns-article/_fandom-all_0__container__']");
```

Comment:
If I get it correctly we don't need anymore `SLOT_SELECTORS` and `IFRAME_SLOT_SELECTORS` since we have now `com.wikia.webdriver.common.contentpatterns.AdSlot`.

Reply:
Will be done later, form the PR description:\
```
Ad slots method rewritten - old methods and selectors to be removed in separate task
```